### DISCUSSION TITLE
Interface Date Format

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12683] [Feature Requests]
We have a Display Date Format, which is fine.
On a form, I want to show the Date Created set as read-only. 
As Date Created is a DateTime it is shown with the Date and the Time.
How to show only the Date.

It would be nice to have an `Interface Date Format` just as we have for the Display.
In that case, I could set the format to Other `MM/dd/yy` to get only the date.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
